src/hooks/useCdpTypes.js

```diff
@@ -1,11 +1,14 @@
-import { useContext } from 'react';
+import { useContext, useCallback } from 'react';
 
 import ilks from 'references/ilkList';
 import { MakerObjectContext } from '../providers/MakerProvider';
 
 export default function useCdpTypes() {
   const { network } = useContext(MakerObjectContext);
-  const cdpTypes = ilks.filter(ilk => ilk.networks.includes(network));
+  const cdpTypes = useCallback(
+    ilks.filter(ilk => ilk.networks.includes(network)),
```

Comment:
this isn't a callback

Reply:
Hmm, using `useCallback` there had *appeared* to solve an issue I was having with too many re-renders when `cdpTypes` was a dependency used [here](https://github.com/makerdao/mcd-cdp-portal/pull/156/files#diff-8b7401be754c41bcc790008654a49821R75). I can't replicate that issue anymore for some reason, and we're no longer using `cdpTypes`.

Reply:
You would use `useMemo` for that purpose. `useCallback` is for when what you're passing out of the hook is a function; `useMemo` is for when it's a value.